nixpkgs/nixpkgs.bzl

```diff
@@ -30,13 +30,19 @@ def _nixpkgs_package_impl(ctx):
   else:
     ctx.template("BUILD", Label("@io_tweag_rules_nixpkgs//nixpkgs:BUILD.pkg"))
 
+  strFailureImplicitNixpkgs = (
+    "You must provide a 'path' or 'repository' or a nix file which point to a"
```

Comment:
Simplify the error message:

> One of 'path' or 'repository' must be provided. The NIX_PATH environment variable is not inherited.

Reply:
Then adjust the error message accordingly.

> One of 'path', 'repository', `nix_file` or `nix_file_content` must be provided. The NIX_PATH environment variable is not inherited.

Error messages should be pithy, to the point and assume as little about what the user is doing as possible.